common/supplemental/supplementalData.xml

```diff
@@ -2604,30 +2604,84 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="es" populationPercent="28"/>	<!--Spanish-->
 		</territory>
 		<territory type="CA" gdp="1774000000000" literacyPercent="99" population="37694100">	<!--Canada-->
-			<languagePopulation type="en" populationPercent="86" officialStatus="official" references="R1194"/>	<!--English-->
```

Comment:
We look for fluent usage, not 'mother tongue'. As near as I can tell from X, the bilingual french-english count is near 18%, making that would be:
english	93%
french	40%

details:

lang | partition | summed
-- | -- | --
french | 22.80% | 40.70%
english | 75.40% | 93.30%
both | 17.90% |  

Reply:
The census data that I used was not just about "mother tongue" but about all languages that are used at home. And while I think that is good data for everything other than French and English, I agree that for those two it would undercount. I will fix the data for those.